cli/command/network/create.go

```diff
@@ -73,7 +77,8 @@ func newCreateCommand(dockerCLI command.Cli) *cobra.Command {
 	flags.VarP(&options.driverOpts, "opt", "o", "Set driver specific options")
 	flags.Var(&options.labels, "label", "Set metadata on a network")
 	flags.BoolVar(&options.internal, "internal", false, "Restrict external access to the network")
-	flags.BoolVar(&ipv6, "ipv6", false, "Enable or disable IPv6 networking")
+	flags.BoolVar(&ipv4, "ipv4", true, "Enable or disable IPv4 address assignment")
```

Comment:
```suggestion
	flags.BoolVar(&ipv4, "ipv4", true, "Enable or disable IPv4 address assignment")
```

Wondering if we should force this to `true` on the CLI side.
The `EnableIPv4` is `*bool` on the API side, so we could just keep it as missing/`null` if user haven't passed this option.
For example if in future we'd need to distinguish between "user explicitly doesn't want ipv4" or "user doesn't care" on the daemon side.

Does that make sense?

Reply:
LOL, I just wrote a blurb below about "defaults".

Reply:
Oh right, I missed that!